pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -583,6 +589,8 @@ def _update_best_and_save(
         if torch.isnan(current):
             current = torch.tensor(float('inf' if self.mode == "min" else '-inf'))
 
+        filepath = self._get_metric_interpolated_filepath_name(ckpt_name_metrics, epoch, step, del_list)
```

Comment:
Why is `del_list` a `list`? It will always have one element if i'm not missing something.

Can you fix it?

Reply:
Great catch! Updated.